src/QirRuntime/build-qir-runtime.ps1

```diff
@@ -2,6 +2,17 @@
 # Licensed under the MIT License.
 
 if ($Env:ENABLE_QIRRUNTIME -eq "true") {
+    Write-Host "##[info]Compile Q# Projects into QIR"
+    $qirStaticPath = Join-Path $PSScriptRoot test QIR-static qsharp
+    dotnet build $qirStaticPath -c $Env:BUILD_CONFIGURATION -v $Env:BUILD_VERBOSITY
+    if ($LastExitCode -ne 0) {
+        Write-Host "##vso[task.logissue type=error;]Failed to compile Q# project at '$qirStaticPath' into QIR."
+        return
+    }
+    Copy-Item -Path (Join-Path $qirStaticPath qir *.ll) -Destination (Split-Path $qirStaticPath -Parent)
```

Comment:
Should we perhaps delete the currently generated QIR files that are in the `/src/QirRuntime/test/QIR-static` folder to avoid confusion?

Reply:
Yes, it would be good to delete the _one_ file that is now being generated (the second file was manually authored and covers functionality not supported by Q# but present in QIR spec)

Reply:
This change does remove that file.